cpp-low-level/Dockerfile

```diff
@@ -17,13 +17,17 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 LABEL io.sunodo.sdk_version=0.1.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 
+ENV MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
 RUN <<EOF
 apt-get update
-apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3
+apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3 wget ca-certificates
+cd /tmp
+wget https://github.com/cartesi/machine-emulator-tools/releases/download/v${MACHINE_EMULATOR_TOOLS_VERSION}/machine-emulator-tools-v${MACHINE_EMULATOR_TOOLS_VERSION}.tar.gz
```

Comment:
Is it possible to use curl instead, and not save the file and just pipe into tar?

Reply:
I was expecting some CHECKSUM from machine-emulator-tools, so I could check the integrity of the tarball before extracting it.

I've opened an issue for that, and will follow the `curl | tar -f -` path for now.